.github/workflows/update-trivy-cache.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Trivy Cache
+
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight UTC
```

Comment:
I think we only want this to run on dev ('default') branch right? Is there a way to make sure other branches don't run this?

Reply:
Good call! I'll add the branch constraint on the job.